### PR TITLE
Ch add testcases 160951497

### PR DESCRIPTION
**Pull request by kafuuma**
## What does this PR do?
This PR delivers all end points belol

## Description of tasks commpleted?
All end pointd below working
```

- POST "/api/v1/auth/signup"
- POST "/api/v1/auth/login"
- GET "/api/v1/menu"
- POST "/api/v1/menu"
- POST "/api/v1/users/orders"
- GET "/api/v1/orders/<int:order_id>"
- GET "/api/v1/users/orders")
- PUT "/api/v1/orders/<int:order_id>"
- GET "/api/v1/order

```
How to test the end points?
```
-clone the repository
-cd into project folder
-pip install -r rqirements.txt
-nosetests --with-coverage -v
```
Relevant piotal tracker stories
#160951497
screenshot from 2018-10-03 16-43-44

### CODE REVIEW

**Review by Kasulejoseph — COMMENTED** [review]
ignore the .pyc file

**Comment by kafuuma** [comment]
thanks @Kasulejoseph 
am going to work on that

File: app/views/add_menu.py [hunk 1]
```diff
@@ -20,13 +22,14 @@ def post(self):
                     try:
                         menu.add_menu_item()
                         if MenuDbQueries().fetch_menu(menu):
-                            return{"response":"menu successfuly created"}
-                        return {"response":"unsuccessful, server error"}
+                            print(MenuDbQueries().fetch_menu(menu))
```
Comment (Sebuliba-Adrian) [inline app/views/add_menu.py:29]: @kafuuma .Do not leave print statements in your code base

File: test/test_menu.py [hunk 2]
```diff
@@ -1,3 +1,38 @@
...
+                    "menu_price": 50000
+                }),
+
+                   headers={
+                       "Authentication": auth_token,
+                       
+                   }
+            )
+
+        print(json.loads(response.data), "strjjghjkl")
```
Comment (Sebuliba-Adrian) [inline test/test_menu.py]: Even here

File: app/views/add_menu.py [hunk 3]
```diff
@@ -7,11 +7,13 @@
 class AddMenu(Resource):
     def post(self):
         user_request = request.get_json()
+        print(user_request)
```
Comment (Sebuliba-Adrian) [inline app/views/add_menu.py:14]: I see print statements here

**Comment by Sebuliba-Adrian** [comment]
Good work @kafuuma . But please increase your test coverage to atleat 75

**Review by Sebuliba-Adrian — CHANGES_REQUESTED** [review]
AndKindly address all the comments

File: app/views/add_menu.py (hunk 3 above)
Comment (kafuuma) [reply app/views/add_menu.py:14]: Alright am going to work on that

File: app/views/add_menu.py (hunk 1 above)
Comment (kafuuma) [reply app/views/add_menu.py:29]: let remove them

File: app/views/add_menu.py (hunk 3 above)
Comment (kafuuma) [reply app/views/add_menu.py:14]: I am planning to run tests transnight 
am sure I will come back tomorrow with 90% coverage

**Comment by MuhweziDeo** [comment]
good file structure

**Comment by MuhweziDeo** [comment]
> ## Pull Request Test Coverage Report for [Build 34](https://coveralls.io/builds/19325697)
> 
>     * **11** of **30**   **(36.67%)**  changed or added relevant lines in **9** files are covered.
> 
>     * **8** unchanged lines in **1** file lost coverage.
> 
>     * Overall coverage increased (+**9.05%**) to **69.466%**
> 
> 
> Changes Missing Coverage 	Covered Lines 	Changed/Added Lines 	%
> [app/models/orders.py](https://coveralls.io/builds/19325697/source?filename=app%2Fmodels%2Forders.py#L16) 	0 	1 	0.0%
> [app/views/signup.py](https://coveralls.io/builds/19325697/source?filename=app%2Fviews%2Fsignup.py#L26) 	4 	5 	80.0%
> [app/views/fectch_all_orders.py](https://coveralls.io/builds/19325697/source?filename=app%2Fviews%2Ffectch_all_orders.py#L20) 	0 	2 	0.0%
> [app/views/fetch_order.py](https://coveralls.io/builds/19325697/source?filename=app%2Fviews%2Ffetch_order.py#L18) 	0 	3 	0.0%
> [app/views/get_menu.py](https://coveralls.io/builds/19325697/source?filename=app%2Fviews%2Fget_menu.py#L18) 	0 	3 	0.0%
> [app/views/update_status.py](https://coveralls.io/builds/19325697/source?filename=app%2Fviews%2Fupdate_status.py#L21) 	0 	4 	0.0%
> [app/views/add_menu.py](https://coveralls.io/builds/19325697/source?filename=app%2Fviews%2Fadd_menu.py#L31) 	4 	9 	44.44%
> Files with Coverage Reduction 	New Missed Lines 	%
> [app/models/db_actions.py](https://coveralls.io/builds/19325697/source?filename=app%2Fmodels%2Fdb_actions.py#L122) 	8 	65.6%
> Totals 	[Coverage Status](https://coveralls.io/builds/19325697)
> Change from base [Build 31](https://coveralls.io/builds/19315627): 	9.05%
> Covered Lines: 	273
> Relevant Lines: 	393
> ##### yellow_heart - [Coveralls](https://coveralls.io)

how do you get this report

**Pull request merged**